apps/web/src/ee/billing/components/SubscriptionProvider.tsx

```diff
@@ -1,18 +1,26 @@
 import React, { useContext } from 'react';
-import { useSubscription } from '../hooks/useSubscription';
+import { useSubscription, UseSubscriptionType } from '../hooks/useSubscription';
+import { ApiServiceLevelEnum } from '@novu/shared';
 
-const SubscriptionContext = React.createContext<{
-  daysTotal: number;
-  daysLeft: number;
-  isLoading: boolean;
-  isFreeTrialActive: boolean;
-  hasPaymentMethod: boolean;
-}>({
-  daysTotal: 0,
-  daysLeft: 0,
+const SubscriptionContext = React.createContext<UseSubscriptionType>({
   isLoading: false,
-  isFreeTrialActive: false,
-  hasPaymentMethod: false,
+  apiServiceLevel: ApiServiceLevelEnum.FREE,
```

Comment:
Do we need these default values, or is `null` just simpler to avoid duplicating the API schema in the FE code?

Reply:
It mostly prevents consumers from having to constantly check if the context value is null or undefined + it can be used also as a fallback